test/controllers/comment_controller_test.exs

```diff
@@ -64,6 +64,28 @@ defmodule CodeCorps.CommentControllerTest do
     test "does not create resource and renders 403 when not authorized", %{conn: conn} do
       assert conn |> request_create(@valid_attrs) |> json_response(403)
     end
+
+    @tag :authenticated
+    test "tracks comment created", %{conn: conn, current_user: current_user} do
```

Comment:
I think we just hadn't refactored the user controller tests yet. Feel free to consolidate, but maybe in a separate PR.